### ISSUE TITLE
Warning in normalizePath with shiny runtime document

### ISSUE THREAD

**Issue by dfalty** [rstudio/rmarkdown#769]
`Warning in normalizePath(path, winslash = winslash, mustWork = mustWork) :
  path[1]="figure-html/unnamed-chunk-1-1.png": No such file or directory`

When Running the following shiny runtime document.

````
---
title: "Untitled"
runtime: shiny
output: html_document

---

```{r}
plot(1)
```
````

**Comment by jjallaire** [comment]
We see this a lot but I always forget why (and whether it would be okay to ignore the warning). @jmcphers Do you recall?

**Comment by jmcphers** [comment]
If you can see the plot in the Shiny app (I could when I ran this), then it's okay to ignore the warning. However this does merit investigation -- if it's unavoidable for this code to generate superfluous errors, we should just explicitly suppress them.

**Comment by yihui** [comment]
Should be fixed via #1171.
